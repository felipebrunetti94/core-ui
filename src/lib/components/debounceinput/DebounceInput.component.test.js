import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/debounceinput/__snapshots__"
});
