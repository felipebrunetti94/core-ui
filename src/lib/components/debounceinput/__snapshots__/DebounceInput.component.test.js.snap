// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots DebounceInput Default 1`] = `
<div>
  <div
    className="sc-debounceinput sc-bdVaJa bDWFJH"
  >
    <input
      onChange={[Function]}
      type="text"
    />
    <label>
      Value: 
      
    </label>
  </div>
</div>
`;
